### PR TITLE
MtxLora. support MtxLora codec

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: src/mtxLora/CommandBinaryBuffer.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import BinaryBuffer from '../utils/BinaryBuffer.js';
+import {extractBits, fillBits} from '../utils/bitSet.js';
+
```
Comment (DarkPark) [inline src/mtxLora/CommandBinaryBuffer.ts:3]:
```suggestion


```

File: src/mtxLora/commands/uplink/MtxCommand.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+/**
+ * Uplink command.
+ *
+ * @example
+ * ```js
+ * import MtxCommand from 'jooby-codec/mtxLora/commands/uplink/MtxCommand.js';
+ *
+ * const command = new MtxCommand({
```
Comment (DarkPark) [inline src/mtxLora/commands/uplink/MtxCommand.ts:13]: uplink example should be reversed

hex -> params

File: src/mtxLora/CommandBinaryBuffer.ts (hunk 1 above)
Comment (Pnom) [reply src/mtxLora/CommandBinaryBuffer.ts:3]: fixed

File: src/mtxLora/commands/uplink/MtxCommand.ts (hunk 2 above)
Comment (Pnom) [reply src/mtxLora/commands/uplink/MtxCommand.ts]: fixed

File: src/mtxLora/commands/downlink/MtxCommand.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ *     fragmentsNumber: 5,
+ *     fragmentIndex: 3,
+ *     data: getBytesFromHex('00 01 02 03 04')
+ * });
+ *
+ * // output command binary in hex representation
+ * console.log(command.toHex());
+ * // 1e 07 02 53 00 01 02 03 04
+ * ```
+ *
```
Comment (DarkPark) [inline src/mtxLora/commands/downlink/MtxCommand.ts:25]:
```suggestion
```

File: src/mtxLora/commands/uplink/MtxCommand.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+ * // output:
+ * {
+ *     sequence: 2,
+ *     last: false,
+ *     fragmentsNumber: 5,
+ *     fragmentIndex: 3,
+ *     data: Uint8Array(3) [0, 1, 2, 3, 4, buffer: ...
+ * }
+ * ```
+ *
```
Comment (DarkPark) [inline src/mtxLora/commands/uplink/MtxCommand.ts:28]:
```suggestion
```

Comment (Pnom) [reply src/mtxLora/commands/uplink/MtxCommand.ts]: fixed

**Review by DarkPark — APPROVED** [review]

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**